src/PlusCommon/PlusCommon.cxx

```diff
@@ -612,6 +622,41 @@ bool vtkPlusCommonExport PlusCommon::IsEqualInsensitive(std::string const& a, st
 }
 
 //----------------------------------------------------------------------------
+bool vtkPlusCommonExport PlusCommon::IsEqualInsensitive(std::wstring const& a, std::wstring const& b)

```

Comment:
Have you tried if you see a difference in loading time with the original version and after this pull request?
Try to load a header with at least a few ten thousand transforms.